tests/SprykerSdkTest/Spryk/Console/SprykRunWithPostSprykArgumentsTest.php

```diff
@@ -36,7 +36,7 @@ public function testExecutesPostSprykWithPredefinedArguments()
         $arguments = [
             'command' => $command->getName(),
             SprykRunConsole::ARGUMENT_SPRYK => 'SprykWithPredefinedArgumentsSubSpryk',
-            '--' . SprykRunConsole::OPTION_INCLUDE_OPTIONALS => ['TemplateWithoutInteraction'],
+            '-i' => ['TemplateWithoutInteraction'],
```

Comment:
For readability, I would also like to keep the long form instead of the short one. We should avoid abbreviations wherever we can.